.github/workflows/debs.yml

```diff
@@ -0,0 +1,44 @@
+# GitHub actions workflow which builds the debian packages.
+
+name: Debs
+
+on:
+  push:
```

Comment:
Do we want to run this on pull requests as well? I'm sometimes quick to dismiss a :x: on a branch because it happens sometimes due to flaky sytests and I don't necessarily bother looking further (because I assume everything that's been PR'd has been :heavy_check_mark:'d on the CI), but I think personally I'd pay more attention if the failure happens on a PR.

Reply:
yeah, I don't really want to tie up 7 GHA workers for 10-12 minutes every time someone pushes to a PR branch. #8555 suggested an approach of building for oldest and newest supported releases, which might make sense, but I'd rather have that as a separate PR.